docs/api-reference/horizon-ai/aggregation/query-dsl/column-expressions.mdx

```diff
@@ -8,24 +8,48 @@ icon: table-columns
 Your Query Instruction can pull data from [Vital resources](/wearables/providers/resources)
 through table column expressions:
 
+**Summaries:**
+
 | Table                                 | Python DSL            | JSON DSL                    
 | ------------------------------------- | --------------------- | ----------------------------
 | Activity daily summary<br/>`activity` | `Activity.col("$1")`  | `{ "activity": "$1" }`
 | Body summary<br/>`body`               | `Body.col("$1")`      | `{ "body": "$1" }`
 | Sleep summary<br/>`sleep`             | `Sleep.col("$1")`     | `{ "sleep": "$1" }`
 | Workout summary<br/>`workout`         | `Workout.col("$1")`   | `{ "workout": "$1" }`
+| Meal summary<br/>`meal`               | `Meal.col("$1")`      | `{ "meal": "$1" }`
+
+**Timeseries:**
+
+| Table                                   | Python DSL                                  | JSON DSL
+| ----------------------------------------| ------------------------------------------- | ----------------------------------------------------
+| Timeseries                              | `Timeseries.col(<resource>).field(<field>)` | `{ "timeseries": "<resource>", "field": "<field>" }`
+
+
+**Shared Source Attributes:**
+
+| Attribute                             | Python DSL                           | JSON DSL
+| ------------------------------------- | ------------------------------------ | -----------------------------------
+| Provider                              | `Source.col("source_provider")`      | `{ "source": "source_provider" }`
+| Source type                           | `Source.col("source_type")`          | `{ "source": "source_type" }`
+| Source App ID (summaries only)        | `Source.col("source_app_id")`        | `{ "source": "source_app_id" }`
+| Workout ID <sup>[1]</sup>             | `Source.col("source_workout_id")`    | `{ "source": "source_workout_id"}`
+| Sport <sup>[1]</sup>                  | `Source.col("source_sport")`         | `{ "source": "source_sport" }`
 
 Each Table Column expression can select one column (a field) of the table (the Vital resource).
 You can use multiple Table Column expressions to select multiple columns in the same query.
 
+<Info>
+<sup>[1]</sup>
+The `source_workout_id` and `source_sport` attributes are only available in the Workout summaries and a few timeseries resource eg. `workout_distance` or `workout_swimming_stroke`
+</Info>
+
 <Info>
 Cross-table query instruction is not supported — you cannot mix Table Column expressions from
 different tables in the same query instruction.
 </Info>
 
 <Note>
-Column expressions for summary types, timeseries types as well as lab testing biomarkers
-are planned to be introduced gradually.
+Column expressions for lab testing biomarkers are planned to be introduced in the future.
 </Note>
```

Comment:
These bits probably should move to the front now that you introduce subsections.